aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -0,0 +1,232 @@
+- The size of name can differ from value.
+- Name should never be scaled bigger than its value.
+
+<NameValueListScaleTable />
+
+<BestPracticeGroup>
+  <Example
+    height="none"
+    bestPractice={{type: 'do', message: 'In this example, names are listed in xsmall size with values displayed in default font-size.'}}
+  >
+    <NameValueListTextGoodExample />
+  </Example>
+  <Example
+    height="none"
+    bestPractice={{type: 'dont', message: `Don’t scale names bigger than their value.`}}
+  >
+    <NameValueListTextBadExample />
+  </Example>
+</BestPracticeGroup>
+
+### Font weight
+
+Making the font bolder or thinner is another way to create hierarchy that’s easily identifiable.
+
+- Typically, use a bold weight on names to increase scanability within the list
+- In some cases, a value may be displayed in a bold weight to draw the reader’s attention. An example of this application might be a dashboard page.
+- Don’t use bold text for both name and value at the same time.
+- Don’t use a bold font-weight for any long format content as this can be very hard to read and overwhelm the user.
+
+  <Example caption="Example: A two column Name-Value List where the names are in a bold font-weight" pad="none" height="none">
+    <NameValueListFontWeightPreview />
+  </Example>
+  <Example caption="Example: A multi column Name-Value List where the values are in a bold font-weight." pad="small" height="none" width="large">
+    <NameValueListFontWeightSecondPreview />
+  </Example>  
+
+<BestPracticeGroup>
+  <Example
+    pad="small"
+    height="none"
+    bestPractice={{type: 'do', message: 'In this example, the bolded names lend to the visual hierarchy of the content displayed and makes it easier to scan.'}}
+  >
+    <NameValueListWeightGoodExample />
+  </Example>
+  <Example
+    pad="small"
+    height="none"
+    bestPractice={{type: 'dont', message: `Don’t use a bold font-weight for both name and value. In this example, the hierarchy and flow of information is not clear which can cause the reader to be overwhelmed by the data presented.`}}
+  >
+    <NameValueListWeightBadExample />
+  </Example>
+</BestPracticeGroup>
+
+<BestPracticeGroup>
+  <Example
+    pad="small"
+    height="none"
+    bestPractice={{type: 'do', message: 'Keep long format content in a regular font-weight (not bold).'}}
+  >
+    <NameValueListBoldExampleGood />
+  </Example>
+  <Example
+    pad="small"
+    height="none"
+    bestPractice={{type: 'dont', message: `Never use bold font-weight to display long format content – this can decrease readability.`}}
+  >
+    <NameValueListBoldExampleBad />
+  </Example>
+</BestPracticeGroup>
+
+### Color
+
+Options for text color include the default `text` <ColorSwatch background="text" /> (default for Name-Value pairs) and `text-strong` <ColorSwatch background="text-strong" /> .
```

Comment:
Is there any guidance on when `text-strong` would be appropriate?

Reply:
I can check with @bahriaditi